python/pyspark/context.py

```diff
@@ -456,7 +463,7 @@ def _dictToJavaMap(self, d):
         jm = self._jvm.java.util.HashMap()
         if not d:
             d = {}
-        for k, v in d.iteritems():
+        for k, v in d.items():
```

Comment:
Will these iteritems() -> items changes have a negative performance impact for Python 2.x users?